consts/version.go

```diff
@@ -2,5 +2,5 @@ package consts
 
 const (
 	SDKIdentifier        = "Go"
-	SDKVersionIdentifier = "3.9.0"
+	SDKVersionIdentifier = "3.12.0"
```

Comment:
should we not just update the version stuff separately from the feature PRs?

Reply:
^ so we gonna revert this or whats the plan? @mehmet-yoti

Reply:
reverted